app/code/Magento/Catalog/Model/Design.php

```diff
@@ -31,11 +31,23 @@ class Design extends \Magento\Framework\Model\AbstractModel
      */
     protected $_localeDate;
 
+    /**
+     * @var \Magento\Framework\TranslateInterface
+     */
+    protected $_translator;
```

Comment:
New variables should be `private` instead on `protected`

Reply:
And should start without underscore `_`